hide/Ide.hx

```diff
@@ -823,6 +823,11 @@ class Ide {
 		return path;
 	}
 
+	public function getUnCachedUrl( path : String ) {
```

Comment:
Could be a static

Reply:
But it uses `getPath()` which is not a static function (uses the `resourceDir` class variable).

Shall I make it static and call `inst.getPath()`?